tessellation/src/path_stroke.rs

```diff
@@ -660,6 +660,44 @@ impl<'l, Output: 'l + GeometryBuilder<Vertex>> StrokeBuilder<'l, Output> {
                 }
             }
 
+            LineJoin::Bevel => {
+            
+                // First we take our initial vertex for compute our triangle
+                let current_vertex = add_vertex!(
+                    self,
+                    Vertex {
+                        position: self.current,
+                        normal: normal,
+                        advancement: self.length,
+                        side: front_side,
+                        }
+                );
+
+                // Compute 2 normal on the right and left side  
+                let normal = if front_side == Side::Right {
+                    vec2(a_line.y, -a_line.x).normalize() * 0.5
+                } else {
+                    vec2(-a_line.y, a_line.x).normalize() * 0.5
```

Comment:
Part of the fix of from #159 was to remove these extra `* 0.5` (which are already applied in the add_vertex macro where needed), so you can remove it here and above.
Also, in that PR `a_line` got renamed into `prev_tangent` and `b_line` into `next_tangent`, and I changed them to be already normalized so you won't need to do it here after rebasing.